server/model.py

```diff
@@ -0,0 +1,70 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, Enum, Date, JSON
+from sqlalchemy.orm import relationship
+
+user = 'mysql'
+password = 'Abc1234!'
+SQLALCHEMY_DATABASE_URL = f"mysql://{user}:{password}@localhost/datedive"
+
+engine = create_engine(
+    SQLALCHEMY_DATABASE_URL
+)
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+Base = declarative_base()
+
+class User(Base):
+    __tablename__ = "User"
+    
+    id = Column(Integer, primary_key=True)
+    age = Column(Integer)
+    gender = Column(Enum('male', 'female'))
+    nickname = Column(String(15))
+    verified = Column(Boolean)
+    signup_date = Column(Date)
+    
+class User_photo(Base):
```

Comment:
Class names should normally use the CapWords convention in python
https://peps.python.org/pep-0008/#class-names